docs/src/components/home-page-editors/component-blocks.tsx

```diff
@@ -0,0 +1,53 @@
+import {
+  InferRenderersForComponentBlocks,
+  component,
+  fields,
+} from '@keystatic/core';
+import { PreviewHeading1 } from './preview-heading-1';
+import { Heading1 } from './heading-1';
+
+export const componentBlocks = {
+  'hilight-heading': component({
+    preview: PreviewHeading1,
+    label: 'Hilight heading',
+    schema: {
+      plainLine1: fields.child({
+        kind: 'block',
+        placeholder: 'Plain text line 1...',
+        formatting: {
+          inlineMarks: 'inherit',
+          softBreaks: 'inherit',
+          listTypes: 'inherit',
+        },
+        links: 'inherit',
+      }),
+      plainLine2: fields.child({
+        kind: 'inline',
+        placeholder: 'Plain text line 2...',
+        formatting: {
+          inlineMarks: 'inherit',
+          softBreaks: 'inherit',
+        },
+        links: 'inherit',
+      }),
+      hilight: fields.child({
+        kind: 'inline',
+        placeholder: 'Hilighted text...',
+        formatting: {
+          inlineMarks: 'inherit',
+          softBreaks: 'inherit',
+        },
+        links: 'inherit',
+      }),
+    },
```

Comment:
Like I mentioned on Slack, it would be great to style the placeholder texts differently (if it's possible?) — they appear very big and very differnt to the actual styles they will be in.

As soon as you type, it's all good, but the _placeholder_ is quite a bit off.

Is there a way to style that? I have never tried personally 😅

Reply:
I totally agree! And maybe this is a case for switching to something like `# Content Management For your **Codebase**`. I can't see a clean way to target the placeholder only, as things are. I'm thinking we'll need to park this and bring it up in the sync. It doesn't feel polished enough to make the homepage as-is 😅